Test/DefaultValues/CreateTableDefaults.hs

```diff
@@ -0,0 +1,84 @@
+{-# OPTIONS_GHC -Wno-ambiguous-fields #-}
+module Test.DefaultValues.CreateTableDefaults where
+{- | This module aims to create some default tables and columns for tables for use 
+elsewhere and make it easier to add fields elsewhere whenever necessary.
+-}
+import IHP.Prelude
+import IHP.IDE.SchemaDesigner.Types
+
+
+{- | Creates a default column where all values are empty lists/text, 
+'Nothing' or 'False'; and for 'ColumnType', the default is 'PUUID'.
+
+Add a new field to the 'Column' type in the 
+"IHP.IDE.SchemaDesigner.Types" file and then 
+set its default value here
+
+Defined as:
+
+@
+defColumn :: Column
+defColumn = Column { 
+                 name = ""
+               , columnType = PUUID 
+               , defaultValue = Nothing
+               , notNull = False 
+               , isUnique = False
+               , generator = Nothing
+               }
+@
+
+If you want a different 'PostgresType' you will need to 
+specify with like so by either using a function:
+
+@setColumnType pgt = defColumn {columnType = pgt}@
+
+Or
+
+Just as part of where you're calling it:
+
+@someDefaultColumnType = defColumn {columnType = PDate}@
+-}
+defColumn :: Column
```

Comment:
As `name` and `columnType` are always set, we could also make these two arguments to this function:

```
defColumn :: Text -> PostgresType -> Column

-- Usage:
defColumn "id" PUUID
```

Reply:
So I was debating with myself for this one as to what the default values were and how best to set them. I tend to have defaults with known constants and then change the values within as necessary -> make new defaults or constants from those, hence the current design. 

But I see the value in your approach and it makes a lot of sense!